sov-modules/sov-modules-api/src/lib.rs

```diff
@@ -119,7 +119,8 @@ pub trait Spec {
         + Into<AddressBech32>
         + Serialize
         + for<'a> Deserialize<'a>
-        + Send;
+        + Send
+        + Display;
```

Comment:
@bkolad @preston-evans98 
Latest addition so addressed can be printed in message. This is change in API so I want ack for this change.